### ISSUE TITLE
error: unknown option '--config'

### ISSUE THREAD

**Issue by s97712** [drizzle-team/drizzle-kit-mirror#423]
When I run `drizzle-kit --config schema/drizzle.config.ts "generate"`,I get the error error: `unknown option '--config'`
The config option can only be placed at the end.

I want to reuse the config option as shown in the example below.

```json5
// package.json
"scripts": {
    "db:kit": "drizzle-kit --config schema/drizzle.config.ts",
    "db:migrate": "pnpm db:kit migrate",
    "db:push": "pnpm db:kit push",
    "db:studio": "pnpm db:kit studio",
}
```
But it seems very difficult.

**Comment by AlexBlokh** [comment]
@s97712 please let me know if that worked?

**Comment by s97712** [comment]
> @s97712 please let me know if that worked?

that not worked

**Comment by AlexBlokh** [comment]
that's fixed, will be in next release

**Comment by AndriiSherman** [comment]
fixed in `drizzle-kit@0.22.0`
